src/pages/ProductListPage/ProductListPage.jsx

```diff
@@ -19,13 +19,15 @@ import "./ProductListPage.css";
 export const ProductListPage = () => {
 
     const [ProductList,setProductList] = useState([]);
+    var products;
```

Comment:
why extra variable here

Reply:
it was temporary I have changed the approach to state